Desktop.Robot.TestApp/Desktop.Robot.TestApp.fsproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
From your PR text, it doesn't seem intentional that you meant to change the target framework and all these versions?

Reply:
Yes, you are right! The changing of the target framework shouldn't have been included into this PR. But I'm always using the latest version of .NET in my projects and I think, this project should also be upgraded (with support for net6.0, net7.0, net8.0). But of course, as you say, this would also need some adaptions to the nuget pipelines and it would be better to do this in a separate PR.